arm-appinsights/2015-05-01/swagger/syntheticMonitor_API.json

```diff
@@ -0,0 +1,402 @@
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "Set of Application Insights synthetic monitor definitions.",
+          "items": {
+            "$ref": "#/definitions/SyntheticMonitor"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The link to get the next part of the returned list of synthetic monitors, should the return set be too large for a single request. May be null."
+        }
+      }
+    },
+    "SyntheticMonitor": {
+      "description": "An Application Insights synthetic monitor definition.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "type": "string",
+          "description": "The kind of web test that this synthetic monitor watches. Choices are ping and multistep.",
+          "default": "ping",
+          "enum": [
+            "ping",
+            "multistep"
+          ],
+          "x-ms-enum": {
+            "name": "WebTestKind",
+            "modelAsString": false
+          }
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/WebTestProperties"
+        }
+      }
+    },
+    "WebTestProperties": {
+      "description": "Metadata describing a synthetic monitor for an Azure resource.",
+      "required": [
+        "Name",
+        "Kind"
+      ],
+      "properties": {
+        "SyntheticMonitorId": {
+          "type": "string",
+          "description": "Unique ID of this WebTest."
+        },
+        "Name": {
+          "x-ms-client-name": "WebTestName",
+          "type": "string",
+          "description": "User defined name if this WebTest."
+        },
+        "Description": {
+          "type": "string",
+          "description": "Purpose/user defined descriptive test for this WebTest."
+        },
+        "Enabled": {
+          "type": "boolean",
+          "description": "Is the test actively being monitored, default is false."
+        },
+        "Frequency": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Interval between test runs for this WebTest."
+        },
+        "Timeout": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Seconds until this WebTest will timeout and fail."
+        },
+        "Kind": {
+          "x-ms-client-name": "WebTestKind",
+          "type": "string",
+          "description": "The kind of web test this is, valid choices are ping and multistep.",
+          "default": "ping",
+          "enum": [
+            "ping",
+            "multistep"
+          ],
+          "x-ms-enum": {
+            "name": "WebTestKind",
+            "modelAsString": false
+          }
+        },
+        "RetryEnabled": {
```

Comment:
is this a required or optional property? If this is optional then what is the default value?